cloudferrylib/os/image/glance_image.py

```diff
@@ -454,3 +457,18 @@ def patch_image(self, backend_storage, image_id):
                 out = json.loads(
                     run("rbd -p images info %s --format json" % image_id))
                 image_from_glance.update(size=out["size"])
+
+    def glance_img_create(self, runner, image_name, image_format, file_path):
+        cfg = self.cloud.cloud_config.cloud
+        out = runner.run(("glance --os-username=%s --os-password=%s --os-tenant-name=%s --os-auth-url=%s " +
```

Comment:
Not sure why external executable is run, instead of python glanceclient object... I would suggest changing this to run pythonic client instead (unless there's some good reason for this call).